### PR TITLE
Bootstrap API

### LINKED ISSUE #7: Bootstrap API
Create the project layout

### PR DESCRIPTION
**Pull request by yoda-yoda**
Bootstrap the API.
closes #7

### CODE REVIEW

File: kulobapp/config/settings.py [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    ),
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'rest_framework.authentication.SessionAuthentication',
+        'oauth2_provider.ext.rest_framework.OAuth2Authentication',
+    ),
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.IsAdmin',
+    ),
+    'DEFAULT_METADATA_CLASS': (
+        'rest_framework.metadata.SimpleMetadata'
```
Comment (ikosen) [inline kulobapp/config/settings.py]: why do we need custom metadata class?

Comment (ikosenn) [inline kulobapp/config/settings.py]: why do we need this?

File: kulobapp/config/settings.py [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+    'DATETIME_FORMAT': 'iso-8601',
+    'DATE_FORMAT': 'iso-8601',
+    'TIME_FORMAT': 'iso-8601',
+}
+
+
+# Database
+# https://docs.djangoproject.com/en/1.9/ref/settings/#databases
+
+DATABASES = {
```
Comment (ikosenn) [inline kulobapp/config/settings.py]: haha... `sqlite`

File: kulobapp/setup.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+version = "0.0.1"
+
+setup(
+    name=name,
+    version=version,
+    packages=find_packages(exclude=['tests', 'tests.*']),
+    description="Slade360 EMR project",
+    long_description=open('README.rst').read(),
+    url="http://pip.slade360.co.ke/docs/{}/".format(name),
```
Comment (ikosenn) [inline kulobapp/setup.py]: hahaha... copy paste noma

File: kulobapp/setup.py [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+name = 'kulobapp_backend'
+
+version = "0.0.1"
+
+setup(
+    name=name,
+    version=version,
+    packages=find_packages(exclude=['tests', 'tests.*']),
+    description="Slade360 EMR project",
```
Comment (ikosenn) [inline kulobapp/setup.py]: hahaha... copy paste noma

File: kulobapp/setup.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+        'django-rest-auth==0.7.0',
+        'stop-words==2015.2.23.1',
+        'wrapt==1.10.6',
+        'sil_transitions==2.1.0a20',
+        'django-redis>=4.0.0,<4.4.0',
+        'ujson==1.34',
+        'celery==3.1.19',
+        'redis==2.10.5',
+        'boto==2.39.0',
+        'python-memcached==1.57',
```
Comment (ikosenn) [inline kulobapp/setup.py]: do we really need all this

File: kulobapp/setup.py [hunk 6]
```diff
@@ -0,0 +1,61 @@
+"""Kulobapp deps."""
```
Comment (ikosenn) [inline kulobapp/setup.py]: i don't think we need this file... There is a similar one below

**Comment by ikosenn** [comment]
good start

File: kulobapp/config/settings.py (hunk 1 above)
Comment (yoda-yoda) [inline kulobapp/config/settings.py]: We won't be using `OPTIONS` ?

File: kulobapp/config/settings.py (hunk 2 above)
Comment (yoda-yoda) [inline kulobapp/config/settings.py]: Haha `psql` inaingia

File: kulobapp/config/settings.py [hunk 7]
```diff
@@ -0,0 +1,187 @@
...
+
+    'DATETIME_FORMAT': 'iso-8601',
+    'DATE_FORMAT': 'iso-8601',
+    'TIME_FORMAT': 'iso-8601',
+}
+
+
+# Database
+# https://docs.djangoproject.com/en/1.9/ref/settings/#databases
+
```
Comment (yoda-yoda) [inline kulobapp/config/settings.py]: haha psql in

**Comment by yoda-yoda** [comment]
@ikosenn done the fixes. Review

File: kulobapp/config/settings.py (hunk 1 above)
Comment (ikosenn) [inline kulobapp/config/settings.py]: cool

File: kulobapp/config/settings.py [hunk 8]
```diff
@@ -0,0 +1,187 @@
...
+    'DEFAULT_PERMISSION_CLASSES': (
+        'rest_framework.permissions.IsAdmin',
+    ),
+    'DEFAULT_METADATA_CLASS': (
+        'rest_framework.metadata.SimpleMetadata'
+    ),
+
+    # Pagination settings
+    'DEFAULT_PAGINATION_CLASS':
+        'rest_framework.authentication.BasicAuthentication',
```
Comment (ikosenn) [inline kulobapp/config/settings.py]: hapa naona umeuma nje...

Comment (yoda-yoda) [inline kulobapp/config/settings.py]: hehe fixed

**Pull request merged**